### ISSUE TITLE
Crash when loading on Windows Server

### ISSUE THREAD

**Issue by szarnyasg** [isaacbrodsky/h3-duckdb#122]
I ran into a crash when loading H3 on Windows Server 2022, running on an `r6i.4xlarge` EC2 instance (16 Intel Xeon cores, 128 GB RAM). To reproduce, run the DuckDB v1.0.0 CLI client:

```batch
C:\Users\Administrator\Downloads\duckdb_cli-windows-amd64>duckdb -cmd "install h3 from community; load h3;"

C:\Users\Administrator\Downloads\duckdb_cli-windows-amd64>echo %errorlevel%
-1073741819
```

The output doesn't contain anything but the error code (which is consistently the same).

**Comment by marklit** [comment]
The above worked for me on Windows 11 so Windows Server might be different enough to warrant its own testing and not great for uncovering Windows 11 issues.

**Comment by isaacbrodsky** [comment]
Does Windows report that the process crashed? If so, it would be really helpful if you could attach a debugger and find where the crash happens.

I presume everything works OK if the SQL you run is anything other than `load h3`?

**Comment by AntoineGiraudKeyrus** [comment]
Hi,

Same issue on Windows 11
![image](https://github.com/user-attachments/assets/2bae7968-0842-48e0-942f-9f0281b02512)

how may i attach debugger to give you more feedback ?

**Comment by AntoineGiraud** [comment]
ps: tried from an other windows 11 computer ... worked fined there ! 🧐
i'll try to reset duckdb on my 1st windows pc & install again h3

PS: congrat's for this great library, can't wait to use it on a side projet on Montreal's bikeshare data 😎

**Comment by AntoineGiraud** [comment]
Problem solved 🦆🚀

thanks to Microsoft Visual C++ Redistributable update
cf. https://github.com/duckdb/duckdb/issues/13848#issuecomment-2343817545

**Comment by carlopi** [comment]
Hi all, this is a variation of this bug: https://github.com/duckdb/duckdb/issues/13848.

I tried to write down what happened and how it can be workarounded or fixed at: https://github.com/duckdb/duckdb/issues/13848#issuecomment-2352674000

If we had connected the dots earlier that might have saved some bugs report, BUT, good think is that now we know what happened AND how to fix it.

I rebuilt `h3` and I am double checking if any other community extension needs rebuilding, but I would consider this solved moving forward.
